discord/embeds.py

```diff
@@ -483,6 +483,44 @@ def set_field_at(self, index, *, name, value, inline=True):
         field['inline'] = inline
         return self
 
+    def insert_field(self, index, *, name, value, inline=True):
+        """Insert a field to the embed object before index.
+
+        The index must point to a valid pre-existing field.
```

Comment:
```suggestion
```

It doesn't, but going out-of-bounds in `list.insert` will just insert at the end so this should be documented.